lib/filesystem/CZipSaver.cpp

```diff
@@ -82,12 +82,11 @@ si64 CZipOutputStream::write(const ui8 * data, si64 size)
 
 ///CZipSaver
 CZipSaver::CZipSaver(std::shared_ptr<CIOApi> api, const boost::filesystem::path & path):
-	ioApi(api),
+	ioApi(std::move(api)),
 	zipApi(ioApi->getApiStructure()),
-	handle(nullptr),
+	handle(zipOpen2_64((const void *) & path, APPEND_STATUS_CREATE, nullptr, &zipApi)),
```

Comment:
What is this? I guess we were lucky and this thing somehow worked before?
```suggestion
	handle(zipOpen2_64(path.c_str(), APPEND_STATUS_CREATE, nullptr, &zipApi)),
```

Reply:
done